src/main/scala/uk/gov/nationalarchives/FileProcessor.scala

```diff
@@ -196,16 +207,19 @@ class FileProcessor(
       fileMetadataChecksum: String,
       bagitTxtChecksum: String,
       manifestSha256Checksum: String,
-      bagInfoChecksum: String
+      bagInfoChecksum: Option[String]
```

Comment:
Maybe this could be named something that implies it's optional, like `potentialBagInfoChecksum` or `possibleBagInfoChecksum`?